hassio_aw2m/config.yaml

```
@@ -0,0 +1,33 @@
+name: AmbientWeather2MQTT
+version: "1.0.0"
+slug: hassio_aw2m
+description: This package listens for local data from Ambient Weather stations (such as the WS-2902C) and converts the incoming data to MQTT events.
+url: "https://github.com/neilenns/ambientweather2mqtt"
+startup: application
+boot: auto
+ports:
+  8132/tcp: 7000
+options:
+  MQTT_SERVER: ""
+  MQTT_USERNAME: ""
+  MQTT_PASSWORD: ""
+  MQTT_REJECT_UNAUTHORIZED: false
+  STATION_MAC_ADDRESS: ""
+  TZ: "America/Chicago"
```

```suggestion
  TZ: "America/Los_Angeles"
```

Aligns with the sample values in the sample `.env` for standalone config.